app/views/layouts/pdf.html.slim

```diff
@@ -0,0 +1,9 @@
+doctype html
+html
+  head
+    title Cashbox
+    = wicked_pdf_stylesheet_link_tag    'application', media: 'all'
```

Comment:
@yunixon I think we need to add separated `pdf.css` file with stylesheets only for PDF.